zeth_contracts/contracts/AltBN128MixerBase_test.sol

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "./AltBN128MixerBase.sol";
+
+
+// Implementation of abstract AltBN128MixerBase contract, to allow testing
+// specific methods.
+contract AltBN128MixerBase_test is AltBN128MixerBase
+{
+    constructor(uint256 mk_depth)
+        public
+        AltBN128MixerBase(mk_depth, address(0), new uint256[](0))
+    {
+    }
+
+    function assemble_public_values_test(uint256 residual_bits)
+        public
+        pure
+        returns (uint256 vpub_in, uint256 vpub_out)
+    {
+        return assemble_public_values(residual_bits);
+    }
+
+    function assemble_hsig_test(
+        uint256[NUM_INPUTS] memory primary_inputs
+    )
+        public
+        pure
+        returns(bytes32 hsig)
+    {
+        return assemble_hsig(primary_inputs);
+    }
+
+    function assemble_nullifier_test(
+        uint256 index,
+        uint256[NUM_INPUTS] memory primary_inputs
+    )
+        public
+        pure
+        returns(bytes32 nf)
+    {
+        return assemble_nullifier(index, primary_inputs);
+    }
+
+    // Dummy implementation of abstract function
+    function verify_zk_proof(
+        uint256[] memory proof,
+        uint256[NUM_INPUTS] memory inputs
```

Comment:
`proofs` and `inputs` are not used which makes the linter complain and causes the CI to fail. Let's either use them in a dummy condition, or switch the linter off for this block.

Reply:
Using the `/* inputs */` approach, similar to C++ seems to satisfy the linter, if that works for you.

Reply:
Oh great, yes that's better.